2.4/root/usr/bin/run-mongod

```diff
@@ -6,20 +6,6 @@ set -o pipefail
 
 source  ${CONTAINER_SCRIPTS_PATH}/common.sh
 
-# Data directory where MongoDB database files live. The data subdirectory is here
-# because mongodb.conf lives in /var/lib/mongodb/ and we don't want a volume to
-# override it.
-export MONGODB_DATADIR=/var/lib/mongodb/data
-
-# Configuration settings.
-export MONGODB_NOPREALLOC=${MONGODB_NOPREALLOC:-true}
-export MONGODB_SMALLFILES=${MONGODB_SMALLFILES:-true}
-export MONGODB_QUIET=${MONGODB_QUIET:-true}
-
-
-export MONGODB_KEYFILE_SOURCE_PATH="/var/run/secrets/mongo/keyfile"
```

Comment:
Seems right to remove it. It was originally added together with the replication example in https://github.com/openshift/mongodb/pull/17, but later its usage was removed in https://github.com/openshift/mongodb/pull/45 and the export was left over. Thanks for spotting it!